tests/system/action/user/test_update.py

```diff
@@ -400,121 +392,350 @@ def test_update_permission_OML_not_high_enough(self) -> None:
+            "user/111",
+            {"group_$_ids": ["1", "4"], "group_$1_ids": [2], "group_$4_ids": [5]},
+        )
 
+    def test_update_permission_group_C_special_2_no_permission(self) -> None:
+        """group C group_$_ids adding meeting in other committee
+        with committee permission for both. Error 403, because touching
+        2 committees requires OML permission
+        """
+        self.permission_setup()
+        self.create_meeting(base=4)
+        self.set_models(
+            {
+                f"user/{self.user_id}": {
+                    "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                    "committee_$_management_level": ["60"],
+                },
+                "user/111": {"group_$_ids": ["1"], "group_$1_ids": [1]},
+            }
+        )
+
+        response = self.request(
+            "user.update",
+            {
+                "id": 111,
+                "group_$_ids": {1: [2], 4: [5]},
+            },
+        )
         self.assert_status_code(response, 403)
         self.assertIn(
-            "You are not allowed to perform action user.update. Missing permissions {'user.can_manage for meeting 4'} or alternative {'CommitteeManagementLevel.CAN_MANAGE for meetings {1}'}.",
+            "You need OrganisationManagementLevel.can_manage_users, because you try to add or remove meetings in Organisation-scope!",
             response.json["message"],
         )
 
-    def test_update_permission_set_2(self) -> None:
-        """ Rights for all field groups,1 one meeting with admin group, other meeting with single right """
-        self.create_meeting()
+    def test_update_permission_group_C_special_3_no_permission(self) -> None:
+        """group C group_$_ids adding meeting in same committee
+        with meeting permission for both. Error 403, because touching
+        2 meetings requires Committee permission
+        """
+        self.permission_setup()
         self.create_meeting(base=4)
-        self.user_id = self.create_user(
-            "test", group_ids=[2, 6]
-        )  # admin-group of meeting/1 and group of meeting 4
-        self.set_group_permissions(6, [Permissions.User.CAN_MANAGE])
-        self.login(self.user_id)
+        self.set_user_groups(self.user_id, [2, 5])  # Admin groups meeting/1 and 4
+        self.set_models(
+            {
+                "meeting/4": {"committee_id": 60},
+                "user/111": {"group_$_ids": ["1"], "group_$1_ids": [1]},
+            }
+        )
+
+        response = self.request(
+            "user.update",
+            {
+                "id": 111,
+                "group_$_ids": {1: [2], 4: [5]},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "You need CommitteeManagementLevel.can_manage permission for committee 60, because you try to add or remove meetings in Committee-scope!",
+            response.json["message"],
+        )
```

Comment:
Please write a simple test, that setting `group_$<some id>_ids` to `[]` correctly removed the `<some id>` from `group_$_ids` and the field `group_$<some id>_ids` is set to none.

Reply:
done, see test_update_remove_group_from_user